### ISSUE TITLE
Z-Axis scaling broken

### ISSUE THREAD

**Issue by HorstBaerbel** [microsoft/SandDance#622]
I have a dataset containing RGB888 colors in range [0,255]. When I choose a scatter plot and enable the z-axis the scaling is NOT like the x- y-axis scaling. I found the slider in settings ("Z scale proportion to Y") and set it to 1, but still the scaling is off.
![grafik](https://user-images.githubusercontent.com/1310747/221896583-b7311f16-0d65-4324-8522-e3d9b09f5ba2.png)
![grafik](https://user-images.githubusercontent.com/1310747/221896780-96a7488c-d3e9-4155-9a73-199ce18c9cc9.png)

**Comment by danmarshall** [comment]
Thanks for reporting. I think there may be a bug for the Scatterplot chart where the z-axis mapping is enabled when it is not in 3d mode. You may just need to click the 3d icon in the top right. Here's screenshots where it's correct for the Column chart:
<img width="863" alt="image" src="https://user-images.githubusercontent.com/11507384/221959818-a5686b85-0bbd-4dc3-98f6-b6a0fcdaf356.png">
<img width="873" alt="image" src="https://user-images.githubusercontent.com/11507384/221959843-c03dacf8-5168-4d3f-adee-5a03cfebd038.png">

**Comment by HorstBaerbel** [comment]
@danmarshall Thanks! That is a workaround, but I'll leave the bug open for now...

**Comment by danmarshall** [comment]
Looking into is, the Z-axis is enabled in 2d mode purposefully - so that users can specify paint order of their scatterplots #512 . This is actually an important feature but seems to have caused some UI confusion that the z-axis is enabled. Perhaps it should be labeled better.

**Comment by danmarshall** [comment]
This link explains why they should be mappable: https://stats.stackexchange.com/questions/11984/how-can-i-remove-the-z-order-bias-of-a-coloured-scatter-plot

**Comment by HorstBaerbel** [comment]
Yes, I understand that. It is the drawing z-order for layering plots. Maybe change the label depending on 2D/3D?

**Comment by danmarshall** [comment]
I think that's a good solution.

**Comment by HorstBaerbel** [comment]
And thanks for the work! Must say I'm pretty new to SandDance, so I might make some mistakes and see everything from the "Dummy perspective" ;)
